scheduler/test/cook/test/mesos/scheduler.clj

```diff
@@ -1328,70 +1327,102 @@
-                 (d/db conn) non-gpu-jobs user->quota user->usage num-considerable nil)))
+                 (d/db conn) non-gpu-jobs user->quota user->usage num-considerable (atom {}) nil)))
         (is (= gpu-jobs
                (sched/pending-jobs->considerable-jobs
-                 (d/db conn) gpu-jobs user->quota user->usage num-considerable nil)))))
+                 (d/db conn) gpu-jobs user->quota user->usage num-considerable (atom {}) nil)))))
     (testing "jobs inside usage quota limited by num-considerable of 3"
       (let [user->usage {test-user {:count 1, :cpus 2, :mem 1024, :gpus 0}}
             user->quota {test-user {:count 10, :cpus 50, :mem 32768, :gpus 10}}
             num-considerable 3]
         (is (= [job-1 job-2 job-3]
                (sched/pending-jobs->considerable-jobs
-                 (d/db conn) non-gpu-jobs user->quota user->usage num-considerable nil)))
+                 (d/db conn) non-gpu-jobs user->quota user->usage num-considerable (atom {}) nil)))
         (is (= gpu-jobs
                (sched/pending-jobs->considerable-jobs
-                 (d/db conn) gpu-jobs user->quota user->usage num-considerable nil)))))
+                 (d/db conn) gpu-jobs user->quota user->usage num-considerable (atom {}) nil)))))
     (testing "jobs inside usage quota limited by num-considerable of 2"
       (let [user->usage {test-user {:count 1, :cpus 2, :mem 1024, :gpus 0}}
             user->quota {test-user {:count 10, :cpus 50, :mem 32768, :gpus 10}}
             num-considerable 2]
         (is (= [job-1 job-2]
                (sched/pending-jobs->considerable-jobs
-                 (d/db conn) non-gpu-jobs user->quota user->usage num-considerable nil)))
+                 (d/db conn) non-gpu-jobs user->quota user->usage num-considerable (atom {}) nil)))
         (is (= gpu-jobs
                (sched/pending-jobs->considerable-jobs
-                 (d/db conn) gpu-jobs user->quota user->usage num-considerable nil)))))
+                 (d/db conn) gpu-jobs user->quota user->usage num-considerable (atom {}) nil)))))
     (testing "jobs inside usage quota limited by num-considerable of 1"
       (let [user->usage {test-user {:count 1, :cpus 2, :mem 1024, :gpus 0}}
             user->quota {test-user {:count 10, :cpus 50, :mem 32768, :gpus 10}}
             num-considerable 1]
         (is (= [job-1]
                (sched/pending-jobs->considerable-jobs
-                 (d/db conn) non-gpu-jobs user->quota user->usage num-considerable nil)))
+                 (d/db conn) non-gpu-jobs user->quota user->usage num-considerable (atom {}) nil)))
         (is (= [job-5]
                (sched/pending-jobs->considerable-jobs
-                 (d/db conn) gpu-jobs user->quota user->usage num-considerable nil)))))
+                 (d/db conn) gpu-jobs user->quota user->usage num-considerable (atom {}) nil)))))
+    (testing "jobs inside usage quota influenced by optimizer"
```

Comment:
I would favor a newline before the `testing` clause

Reply:
Done.